migrate/20250225_add_loadbalancer_ports.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:load_balancer_port) do
+      uuid :id, primary_key: true
+      uuid :load_balancer_id, null: false
+      Integer :src_port, null: false
+      Integer :dst_port, null: false
+      String :health_check_endpoint, text: true, null: false, default: "/up"
+      Integer :health_check_interval, null: false, default: 30
+      Integer :health_check_timeout, null: false, default: 15
+      Integer :health_check_up_threshold, null: false, default: 3
+      Integer :health_check_down_threshold, null: false, default: 2
+      String :health_check_protocol, text: true, null: false, default: "http"
+
+      foreign_key [:load_balancer_id], :load_balancer, key: :id
+      index [:load_balancer_id, :src_port, :dst_port], unique: true, name: :lb_port_unique_index
+
+      check Sequel.lit("health_check_timeout > 0"), name: :chk_health_check_timeout
+      check Sequel.lit("health_check_interval > 0 AND health_check_interval < 600"), name: :chk_health_check_interval
+      check Sequel.lit("health_check_timeout <= health_check_interval"), name: :chk_timeout_interval
+      check Sequel.lit("health_check_up_threshold > 0"), name: :chk_health_check_up_threshold
+      check Sequel.lit("health_check_down_threshold > 0"), name: :chk_health_check_down_threshold
+    end
+
+    create_table(:load_balancer_vm_port) do
+      uuid :id, primary_key: true
```

Comment:
are we adding different port assignments to different VMs in that same load balancer? Again, how different is this then to have all together multiple load balancers per vm?